src/main/java/co/pragra/attendancesystem/service/CohortService.java

```diff
@@ -0,0 +1,58 @@
+package co.pragra.attendancesystem.service;
+
```

Comment:
This file is not needed anymore.

We are now using interface that extends JpaRepository. 

@Repository
public interface CohortRepo extends JpaRepository<Cohort, Long> {
}

Reply:
Ok will remove the service layer.